tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore2IastTests.cs

```diff
@@ -37,7 +37,7 @@ public AspNetCore2IastTestsTwoVulnerabilityPerRequestIastEnabled(AspNetCoreTestF
     public abstract class AspNetCore2IastTestsVariableVulnerabilityPerRequestIastEnabled : AspNetCore2IastTests
     {
         public AspNetCore2IastTestsVariableVulnerabilityPerRequestIastEnabled(AspNetCoreTestFixture fixture, ITestOutputHelper outputHelper, int vulnerabilitiesPerRequest)
-            : base(fixture, outputHelper, enableIast: true, testName: "AspNetCore2IastTestsEnabled", isIastDeduplicationEnabled: true, samplingRate: 100, vulnerabilitiesPerRequest: vulnerabilitiesPerRequest)
+            : base(fixture, outputHelper, enableIast: true, testName: "AspNetCore2IastTestsEnabled", isIastDeduplicationEnabled: false, samplingRate: 100, vulnerabilitiesPerRequest: vulnerabilitiesPerRequest)
```

Comment:
Unclear to me why we turn duplication off all over the place in this test

Reply:
maybe could be in a comment? does it mean you also need to turn deduplication off  for some frameworks in production?

Reply:
This is only affects the integration tests. In production, in the worst case, there could potentially be a situation where there are two vulnerabilities in the same method with the same vulnerability kind and only one would be reported, but this issue has already discussed with the IAST team and is the expected behavior